frontend/packages/yki/src/redux/reducers/reservation.ts

```diff
@@ -1,16 +1,18 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { APIResponseStatus } from 'shared/enums';
 
-import { ReservationRequest } from 'interfaces/reservation';
+import {
+  ReservationErrorCause,
+  ReservationRequest,
+} from 'interfaces/reservation';
 
 interface ReservationState extends Partial<ReservationRequest> {
   status: APIResponseStatus;
-  emailAlreadyQueued: boolean;
+  errorCause?: ReservationErrorCause;
```

Comment:
Vaikuttais, että tämä jää jossain kohtaa resetoimatta. Jos ilmoittautuu jonoon, se tuottaa virheen ja sitten menee ilmoittautumaan jonoon johonkin toiseen niin käyttöliittymä näyttää virhedialogin heti.

Reply:
Nyt näyttäis toimivan oikein. En löytäny muita ongelmia eli eiköhän tän voine mergeillä.